.travis.yml

```diff
@@ -1,6 +1,10 @@
 sudo: false
 language: php
 
+env:
+  global:
+    - secure: "foo"
```

Comment:
@bighappyface 

Alternatively, you can - of course - specify a `GITHUB_TOKEN` in Travis!